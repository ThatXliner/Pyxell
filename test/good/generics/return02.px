
func f<T>(Bool b) T def
    if b do
        return 1e0
    else do
        return -10^^18

print f(true), f(false)
