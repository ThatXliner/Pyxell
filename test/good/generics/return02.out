1 -1e+18
